### PR TITLE
buffer: add writeInt32LE impl and docs

### PR DESCRIPTION
**Pull request by yorkie**
Implements the function `buffer.writeInt32LE()` which [Rokid/node-turen](https://github.com/Rokid/node-turen) depends on it.

/cc @legendecas @LanFly

### CODE REVIEW

**Comment by yorkie** [comment]
This fixes #163.

File: src/js/buffer.js [hunk 1]
```diff
@@ -302,6 +302,22 @@ Buffer.prototype.writeUInt32LE = function(value, offset, noAssert) {
...
+  offset = offset >>> 0;
+  if (!noAssert)
+    checkInt(this, value, offset, 4, 0x7fffffff, -0x80000000);
+  this._builtin.writeUInt8(value & 0xff, offset);
+  this._builtin.writeUInt8((value >>> 8) & 0xff, offset + 1);
+  this._builtin.writeUInt8((value >>> 16) & 0xff, offset + 2);
+  this._builtin.writeUInt8((value >>> 24) & 0xff, offset + 3);
+  return offset + 4;
+};
+
```
[the remark this answers is not in the document]
Comment (yorkie) [reply src/js/buffer.js]: @algebrait Could we open another PR or issue to make this improvements or discussion? This causes rewriting all the `write*` methods.

The principle of current implementation exposes `writeUInt8` to write 1 byte, and implement other methods in pure JavaScript.

[the remark this answers is not in the document]
Comment (algebrait) [reply src/js/buffer.js]: reasonable

**Comment by yorkie** [comment]
Ping @legendecas :)

**Pull request merged**